src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,37 @@
-export const TodoInfo = () => {};
+import cn from 'classnames';
+import { FC } from 'react';
+import { UserInfo } from '../UserInfo';
+import { ToDo } from '../../types/types';
+
+type Props = {
```

Comment:
You've already started nicely to export typing to separate files, so it would be cool to continue here as well, and export the type for the `TodoInfo` component to the types.tsx file and give this type a more convenient name (For example: `TodoInfoType`).